weasyl/moderation.py

```diff
@@ -304,10 +307,35 @@ def finduser(userid, form):
             lo.c.email.op('~')(form.email),
             lo.c.email.op('ilike')('%%%s%%' % form.email),
         ))
-    else:
+
+    # Filter for banned and/or suspended accounts
+    if form.excludeactive == "on":
+        q = q.where(d.sa.or_(
```

Comment:
`~` being a regex match, you can simplify this to `q.where(lo.c.settings.op('~')('[bs]'))`.